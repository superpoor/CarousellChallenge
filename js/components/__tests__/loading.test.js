import React from 'react';
import renderer from 'react-test-renderer';

import Loading from '../loading'

it('renders correct Loading component', () => {
  const rendered = renderer.create(
    <Loading />
  ).toJSON();
  expect(rendered).toMatchSnapshot();
});