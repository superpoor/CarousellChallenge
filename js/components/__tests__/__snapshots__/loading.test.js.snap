// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correct Loading component 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <ActivityIndicator
    animating={true}
    color="#999999"
    hidesWhenStopped={true}
    size="large"
  />
</View>
`;
